corona-warn-app/cwa-app-android
# Unresponsive UI after continuous certificate deletion (EXPOSUREAPP-13660)

[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13660)


How to test:

Using the certificates in the ticket:

1. Scan the first certificate, on the details page, click the three dot menu up top and delete the cert. 
2. Scan the second certificate (go straight to the scanner without traveling to any other screen between the first scan and the second one) -> go to the three dot menu up top and delete the cert -> you should land on the home fragment and the UI should still be responsive.
